bokehjs/src/coffee/renderer/glyph/text.coffee

```diff
@@ -21,15 +25,10 @@ class TextView extends Glyph.View
 
   draw_legend: (ctx, x1, x2, y1, y2) ->
     ctx.save()
-    reference_point = @get_reference_point()
-    if reference_point?
-      glyph_settings = reference_point
-    else
-      glyph_settings = @props
-    text_props = @visuals.text
-    text_props.set(ctx, glyph_settings)
+    @text_props.set_value(ctx)
     # override some features so we fit inside the legend
-    ctx.font = text_props.font(12)
+    ctx.font = @text_props.font_value()
+    ctx.font = ctx.font.replace(/\b[\d\.\d]+[\w]+\b/, '10pt')
```

Comment:
Can't hurt, but superfluous (so it might confuse someome). It will match "3.5pt" without it (square brackets imply no order)

<img width="217" alt="screen shot 2015-08-10 at 10 02 38 pm" src="https://cloud.githubusercontent.com/assets/1078448/9189250/a4efdf96-3fab-11e5-9ad9-da994a8d1dcb.png">